### ISSUE TITLE
Allow Turso remote databases

### ISSUE THREAD

**Issue by moltar** [promptfoo/promptfoo#1759]
It appears that the `turso` driver is already used, but because the credentials are hard-coded to be read from the file, it is not possible to use Turso hosted databases.

Would it be possible to modify this config to read config from the environment first, and then fallback to the local database?

https://github.com/promptfoo/promptfoo/blob/1b55a15e8019d481cd080ce9d3a151a35ef113d8/drizzle.config.ts#L4-L12

**Comment by mldangelo** [comment]
Hi @moltar, generically overriding the database path and credentials would be a great addition to promptfoo, and we’d welcome a PR to implement this change. If you decide to proceed, please also update [`src/envars.ts`](https://github.com/promptfoo/promptfoo/blob/main/src/envars.ts) and the [self-hosting documentation](https://www.promptfoo.dev/docs/usage/self-hosting/#eval-storage) to include the new environment variables. Let me know if you have any questions!

**Comment by mldangelo** [comment]
@moltar would you like help getting this in?

**Comment by moltar** [comment]
Yeah, if you provide a bit more guidance, I can definitely contribute! Basically just where things should go and the expectations on the PR. Thanks.
